source/routing/defining-your-routes.md

```diff
@@ -301,13 +301,13 @@ Router.map(function() {
 ```
 
 Like all routes with a dynamic segment, you must provide a context when using a `{{link-to}}`
-or `transitionTo` to programatically enter this route.
+or `transitionToRoute` to programatically enter this route.
 
 ```app/routes/application.js
 export default Ember.Route.extend({
   actions: {
     error: function() {
-      this.transitionTo('catchall', 'application-error');
+      this.transitionToRoute('catchall', 'application-error');
```

Comment:
Yes, you're right. This can be declined. 

Sent from my iPhone

> On Apr 5, 2015, at 1:15 AM, Eric Kelly <notifications@github.com> wrote:
> 
> In source/routing/defining-your-routes.md:
> 
> >  
> >  ```app/routes/application.js
> >  export default Ember.Route.extend({
> >    actions: {
> >      error: function() {
> > -      this.transitionTo('catchall', 'application-error');
> > +      this.transitionToRoute('catchall', 'application-error');
> AFAIK transitionToRoute() is only available on controllers. Was this changed somewhere?
> 
> —
> Reply to this email directly or view it on GitHub.
>